### ISSUE TITLE
`beginNextRound` constains assumption on recieving local messages

### ISSUE THREAD

**Issue by Kubuxu** [filecoin-project/go-f3#351]
https://github.com/filecoin-project/go-f3/blob/ad199a817719535579eff6092ce9d66f00171b9a/gpbft/gpbft.go#L854-L858

@adlrocha observes a panic from GPBFT in that piece of code.
Commit might require a similar treatment as converge: https://github.com/filecoin-project/go-f3/pull/334

Part of https://github.com/filecoin-project/go-f3/issues/319

**Comment by anorth** [comment]
I don't think this one is an assumption on receiving local messages because I don't think we can reach beginNextRound without receiving some message.

**Comment by masih** [comment]
> I don't think this one is an assumption on receiving local messages because I don't think we can reach beginNextRound without receiving some message.

You are right. Thanks Alex. 

The bug that caused this in @adlrocha was related to missing power table values unrelated to the core protocol. I think we can close this issue. I'll close my PR.
